packages/core/package.json

```diff
@@ -3,6 +3,15 @@
   "version": "4.3.0",
   "private": false,
   "description": "V-validation core package",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
+    }
+  },
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
```

Comment:
The CJS packaging definitions should probably be removed when building ESM-only package